consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/statemachine/RoundChangeManager.java

```diff
@@ -48,7 +48,7 @@
 
     // Store only 1 round change per round per validator
     @VisibleForTesting
-    final Map<Address, SignedData<RoundChangePayload>> receivedMessages = Maps.newHashMap();
+    final Map<Address, SignedData<RoundChangePayload>> receivedMessages = Maps.newLinkedHashMap();
```

Comment:
What's the reason for using the LinkedHashMap?

Reply:
To maintain insertion order - its not _required_ - but it makes testing and debug a whole lot easier